charts/govuk-jobs/templates/govuk-mirror-sync-cronjob.yaml

```diff
@@ -45,11 +45,11 @@ spec:
               workingDir: /data
               resources:
                 limits:
-                  cpu: 2
-                  memory: 8000Mi
+                  cpu: 4
+                  memory: 10000Mi
                 requests:
-                  cpu: 1
-                  memory: 2000Mi
+                  cpu: 2
+                  memory: 5000Mi
```

Comment:
Wow. Room for optimisation at some point I guess! (I agree with just chucking resources at it for now.)

Reply:
Yeah 😢  - need to have another look at why it's gobbling so much up.